### PR TITLE
CB-4548 Cloudbreak and FreeIPA must persist the minaSshdServiceID used and use that for service endpoint lookup

### PR DESCRIPTION
**Pull request by lacikaaa**

### CODE REVIEW

File: freeipa/src/main/resources/schema/app/20191203144540_CB-4548_alter_stack_add_column_minasshdserviceid.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+-- // CB-4548 alter stack add column minasshdserviceid
+-- Migration SQL that makes the change goes here.
```
Comment (keyki) [inline freeipa/src/main/resources/schema/app/20191203144540_CB-4548_alter_stack_add_column_minasshdserviceid.sql]: There is 2 sql files with the same content.

File: ccm-connector/src/main/java/com/sequenceiq/cloudbreak/ccmimpl/altus/GrpcMinaSshdManagementClient.java [hunk 2]
```diff
@@ -228,7 +229,7 @@ public MinaSshdService acquireMinaSshdServiceAndWaitUntilReady(String requestId,
             Supplier<CcmException> timeoutExceptionSupplier = () -> new CcmException(String.format("Timed out while trying to %s", actionDescription), true);
 
             return RetryUtil.performWithRetries(
-                    () -> client.unregisterSshTunnelingKey(requestId, DUMMY_MINASSHD_SERVICE_ID, keyId), actionDescription,
+                    () -> client.unregisterSshTunnelingKey(requestId, minaSshdServiceId, keyId), actionDescription,
```
Comment (keyki) [inline ccm-connector/src/main/java/com/sequenceiq/cloudbreak/ccmimpl/altus/GrpcMinaSshdManagementClient.java]: DUMMY_MINASSHD_SERVICE_ID can be removed

**Comment by keyki** [comment]
One minor thing otherwise LGTM

File: ccm-connector/src/main/java/com/sequenceiq/cloudbreak/ccmimpl/altus/GrpcMinaSshdManagementClient.java (hunk 2 above)
Comment (lacikaaa) [reply ccm-connector/src/main/java/com/sequenceiq/cloudbreak/ccmimpl/altus/GrpcMinaSshdManagementClient.java]: thanks, removed

**Pull request merged**